tests/test_deployments.py

```diff
@@ -738,6 +738,31 @@ def test_custom_flow_run_names(self, test_deployment, use_hosted_orion):
 
         assert flow_run.name == "a custom flow run name"
 
+    def test_accepts_tags(self, test_deployment):
+        d, deployment_id = test_deployment
+
+        flow_run = run_deployment(
+            f"{d.flow_name}/{d.name}",
+            tags=["I", "love", "prefect"],
+            timeout=0,
+            poll_interval=0
+        )
+
+        assert sorted(flow_run.tags) == ["I", "love", "prefect"]
+
+
+    def test_accepts_idempotency_key(self, test_deployment):
+        d, deployment_id = test_deployment
+
+        flow_run = run_deployment(
+            f"{d.flow_name}/{d.name}",
+            idempotency_key="12345",
+            timeout=0,
+            poll_interval=0
+        )
+
+        assert flow_run.idempotency_key == "12345"
```

Comment:
Can you call `run_deployment` again with the same idempotency key and assert the flow run ids match?

Reply:
Ah, the above test isn't really doing anything meaningful. Thanks for the input -- updated accordingly.